x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+API_KEY_ENCODED=$1
+API_ENDPOINT=$2
+
+updateStepProgress() {
+  echo "  GET $API_ENDPOINT/step/$1?status=$2"
```

Comment:
This means Kibana has to be accessible for this to work. This is ok but we should mention it in our docs or when users hit erros, make sure we provide good error messages that Kibana is not reachable for example.